cni/network/network_windows_test.go

```diff
@@ -732,6 +732,24 @@ func TestGetNetworkNameSwiftv2FromCNS(t *testing.T) {
 			want:    parsedMacAddress,
 			wantErr: false,
 		},
+		{
+			name: "Unhappy path: Get Network Name from CNS for swiftv2 AccelnetNIC with empty interfaceInfo",
+			plugin: &NetPlugin{
+				Plugin:      plugin,
+				nm:          network.NewMockNetworkmanager(network.NewMockEndpointClient(nil)),
+				ipamInvoker: NewMockIpamInvoker(false, false, false, false, false, true, false),
+				report:      &telemetry.CNIReport{},
+				tb:          &telemetry.TelemetryBuffer{},
+			},
+			netNs: "azure",
+			nwCfg: &cni.NetworkConfig{
+				CNIVersion:   "0.3.0",
+				MultiTenancy: false,
+			},
+			interfaceInfo: &network.InterfaceInfo{},
+			want:          parsedMacAddress,
+			wantErr:       true, // should return empty network name
```

Comment:
After running this test it doesn't seem to pass on my computer locally? The network name is empty because I believe it goes through !nwCfg.Multitenancy path when nwCfg Name is an empty string but doesn't return an error.